# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1251894600 -7200
#      Wed Sep 02 14:30:00 2009 +0200
# Node ID 4dd8fc7c106c6177bb11d5833f70e409794af9dc
# Parent  8bea4e89326f4f59e8630f8432538c3f26eb839f
ChangeLog fix for last change: FLOAT_STORE->FLOAT_TRUNCATE

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,11 +1,11 @@
 2009-09-02  Jaroslav Hajek  <highegg@gmail.com>
 
-	* configure.in (FLOAT_STORE): New config macro.
+	* configure.in (FLOAT_TRUNCATE): New config macro.
 
 2009-09-02  Jaroslav Hajek  <highegg@gmail.com>
 
 	* acx_blas_f77_func.m4 (ACX_BLAS_F77_FUNC): Check for correct INTEGER
 	size.
 
 2009-09-01  John W. Eaton  <jwe@octave.org>
 
diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,11 +1,11 @@
 2009-09-02  Jaroslav Hajek  <highegg@gmail.com>
 
-	* oct-cmplx.h: Rewrite the comaprison ops. Use FLOAT_STORE.
+	* oct-cmplx.h: Rewrite the comaprison ops. Use FLOAT_TRUNCATE.
 
 2009-09-01  Jaroslav Hajek  <highegg@gmail.com>
 
 	* oct-cmplx.h: Correct strict operators in macros.
 
 2009-08-31  John W. Eaton  <jwe@octave.org>
 
 	* liboctave/lo-ieee.cc (octave_ieee_init): Abort if floating point
